mne_connectivity/viz/circle.py

```diff
@@ -141,14 +141,32 @@ def plot_connectivity_circle(con, node_names, indices=None, n_lines=None,
                 subplot = (subplot,)
             ax = plt.subplot(*subplot, polar=True)
 
-    return _plot_connectivity_circle(
-        con=con, node_names=node_names, indices=indices, n_lines=n_lines,
-        node_angles=node_angles, node_width=node_width,
-        node_height=node_height, node_colors=node_colors,
-        facecolor=facecolor, textcolor=textcolor,
-        node_edgecolor=node_edgecolor, linewidth=linewidth,
-        colormap=colormap, vmin=vmin, vmax=vmax, colorbar=colorbar,
-        title=title, colorbar_size=colorbar_size, colorbar_pos=colorbar_pos,
-        fontsize_title=fontsize_title, fontsize_names=fontsize_names,
-        fontsize_colorbar=fontsize_colorbar, padding=padding, ax=ax,
-        interactive=interactive, node_linewidth=node_linewidth, show=show)
+    try:
```

Comment:
this is really a code smell. Please avoid try/except blocks like this and try to catch errors explicitly as early as possible

Reply:
The issue is that the code is in `mne-python` and there is no way to undo the default arguments since they are passed since if you pass them to `matplotlib` they will throw exceptions for `None`.

Maybe I will just leave the small colorbar as default in the other PR but just remove it from gridspec and then we don't have to do this.